### ISSUE TITLE
Revisit FAQ and add new 

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#1604]
Let's revisit our current FAQ!
Since our FAQ is linkable and addressable and you can direct users there with onboarding

### Spec
- remove obsolete questions
- update current/present questions
- add frequent questions from discord, so we can redirect people there and save time! :)

https://github.com/kodadot/nft-gallery/blob/main-nuxt/langDir/en.json

![image](https://user-images.githubusercontent.com/5887929/147408129-9b8e6d4b-8ab3-4404-9efc-3611323ec997.png)

**Comment by vikiival** [comment]
May use Markdown FAQ?

[@nuxt/content](https://content.nuxtjs.org)

**Comment by cryptodamsky** [comment]
Since i can't really edit the code yet, i'll atleast have full list of how should the FAQ look. It's also necessary to update all the links and make the tutorials for polkadot js wallet, obtaining KSM and how to simple mint along when the new FAQ is going to be published. I'm on it

**Comment by kkukelka** [comment]
@cryptodamsky any updates on this? I would be glad to implement your updated FAQ items. Just give me a heads-up when you are ready

**Comment by cryptodamsky** [comment]
"1": "What are NFTs?"
"NFT or non-fungible tokens refer to unique assets, which are stored on blockchain and maintain unique ownership."

"2": "What is minting?"
"Minting is a process in which your cryptocurrency wallet and blockchain communicate together and make you able to create nfts."

"3": "How can i mint nfts?"
"In the left upper corner, you click on "[create](https://kodadot.xyz/rmrk/create)" and choose simple or classic mint. If you don't know how to continue, you can watch this tutorial."

"4": "What's the flow when I'm NFT creator?"
"First of all, you need to install [Polkadot Extension](https://polkadot.js.org/extension), **backup your seed**. Then you can top up your wallet address [here](https://kodadot.xyz/rmrk/credit) or use our [grant](https://tally.so/r/mVP06w) feature. Once you have funds, head to [Create](https://kodadot.xyz/#/rmrk/create), choose Collection. Wait till Collection is minted, then you go to [Create Token](https://kodadot.xyz/#/rmrk/create) and Mint your very first token! Feel free to share it with us on our [discord](https://discord.gg/gHrQTJam)."

"5": "How can i sell KSM i got for my art?"
"Right now, there's plenty of [exchanges](https://www.coingecko.com/en/coins/kusama#markets) which support KSM trading. The safest are probably [Binance](https://www.binance.com/en) and [Bybit](https://www.bybit.com/en-US/) or try decentralized [simpleswap](https://simpleswap.io/?ref=6cc6e0c538ad)."

"6": "How can i obtain KSM as new-comer?"
"We recommend to obtain KSM through custodial exchanges like [Binance](https://www.binance.com/en) or [Bybit](https://www.bybit.com/en-US/) or you can use our RAMP network - a fiat ramp [here](https://kodadot.xyz/rmrk/credit). If you are struggling with this, feel free to ask for a [grant](https://tally.so/r/mVP06w) or let us know in "[help](https://discord.gg/AkCJPmwJ)" channel on discord."

"7": "Can I use this in virtual worlds?"
"We are exploring possibilities, we actually plan to actively research on this in our milestone 2."

"8": "How can i buy KSM with ETH, DAI, BTC..?"
"For best rates, you can [use custodial exchange from list](https://www.coingecko.com/en/coins/kusama). For best experience [we recommend you to use Binance](https://www.binance.com/en/register?ref=T1JB89F0). If you are [looking for simple trade, try SimpleSwap](https://simpleswap.io/?ref=6cc6e0c538ad)."

"9": "I've got errors, how to resolve them?"
"We are trying to deliver incremental updates on daily basis & fixes ASAP. We may have published new version and your browser has cached old one. If you are on **Windows**, try hold down **Ctrl and then press F5**. **Mac users**, hold down **CMD and Shift and then press R**. If this did not help, DM us on Twitter or discord with screenshot where is developer console open."

"10": "What does SEND, CONSUME, LIST mean?"
"SEND will send it to someone, CONSUME will burn it, LIST will list it for sale so someone can buy it from you."

"11": "I've minted NFT, but I don't see it in gallery, where is it?"
"Best way is to check through [Subscan Explorer](https://kusama.subscan.io/) and lookup your address. [You can find example how minted NFT looks through explorer, when you look on extrinsics.](https://kusama.subscan.io/extrinsic/6095478-1)"

"12": "Is there any other way to obtain small amount of KSM?"
"Yes, if for example it's problematic for you to withdraw from an exchange or use fiat-onramp, you may visit our discord and ask for small amount of KSM in "[help](https://discord.gg/AkCJPmwJ)" channel on our discord or ask for [grant](https://tally.so/r/mVP06w) to cover minting costs."

"13": "How can i sell my nft?"
"If you haven't set the price of your nft at the beginning, navigate yourself to your [profile](https://kodadot.xyz/rmrk/u/), choose your NFT in "Created" and click on LIST, which will make you able to set the price"

"14": "Royalties streams settings?"
"Right now, we don't have chain logic, but we plan to implement something great, soon. [Stay with us and follow @KodaDot!](https://twitter.com/kodadot)."

"15": "Why did you choose to build on Kusama/Polkadot and not Ethereum?"
"We believe that costs for actions at Ethereum may be high in future and its current PoW is not environmentally friendly. We believe in network effects of Kusama/Polkadot effects from other parachains. A more detailed answer can be found [here](https://github.com/kodadot/nft-gallery/issues/529)."

"16": "What's the difference between creating Classic, Simple and Permafrost?"
"CLASSIC" let's you create collection and add NFTs to it. "SIMPLE" mint will create NFT collectible and add it to your profile and "PERMAFROST" is nft minting using **Arweave** protocol: The protocol matches people who have hard drive space to spare with those individuals and organizations that need to store data or host content permanently."

"17": "Are you funded?"
"Right now, we have [Kusama Treasury funding for Milestone 1](https://kodadot.xyz/rmrk/@.common:links.kusama_polkassembly.href). We are actively looking for funding options, eventually we would raise private funds. DMs us, if you are interested into private sale."

"18": "Do you have Telegram channel?",
"We have only private channel for NFT creators, DM us if you are NFT creator!"

"19": "Will you have AR/VR mode or implementation?"
"Yes, that's part of our Milestone 2, stay tuned!"

"20": "How to open developer console?"
"If you are on Mac, on Chrome hit **Option, CMD, J**, Firefox **Command + Option + K**."

"21": "How do I update metadata for extension?"
"Go to [Dotapps.io, Settings > Metadata](https://cloudflare-ipfs.com/ipns/dotapps.io/?rpc=wss%3A%2F%2Fkusama-rpc.polkadot.io#/settings/metadata) and see there button **Update metadata**, then refresh your window, should be working with recent chain types."

"22": "I've got \"Low balance error\""
"Check if you have **transferable** $KSM on your account, not staked. That's different. You can check it at [Dotapps.io > Comptes](https://cloudflare-ipfs.com/ipns/dotapps.io/?rpc=wss%3A%2F%2Fkusama-rpc.polkadot.io#/accounts) and at balances unfold, you can see there **Transferable** amount."

"23": "How do I set identity?"
"You can learn more [how to set identity on Kusama network](https://guide.kusama.network/docs/learn-identity)."

"24": "Is there an official KodaDot token?" 
"No, there is no official token launched. Please follow official social media and be aware of scam and phissing attempts from unofficial sources. DYOR."

"25": "How can I contribute to the project?" 
"We are always welcoming contributions from our community. There are numerous [open issues](https://github.com/kodadot/nft-gallery/issues) for all skill levels - some will even be rewarded in $KSM. [Make your first contribution!](https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md)"

"26": "Is it possible to move nfts frome one collection to another?"
"No, this possibility is not here at this moment, if you would like to propose a change, feel free to create [issue](https://github.com/kodadot/nft-gallery/issues) on our github"

This is what i would propose as our updated FAQ, but there is some stuff we need to agree on.

- I wrote to ramp network, to find out when buying KSM with fiat is going to be avalaible and they don't know. Question is, if it wouldn't be better to just not link it now and link it when it's avalaible again, or, when there's some functional dex on KSM, so people can exchange dot for ksm (dot is avalaible).
- As i tried to avoid it, link to ramp network is only in question number 6 for now.

I'll also be making tutorials on most of these things, like how to create polkadot js wallet, how to simple mint, how to obtain KSM, how to sell your art, show several functions, what can one see in KodaDot and so on. So these questions will be updated and be ending with "if you have problems with this, you can watch this tutorial". But that will be once the tutorials are ready and proofed. 

I guess we can then later continue in this issue and once the tutorial is ready on youtube i'll just send updated question number and we can update it. Also i would like to link directly to a playlist on KodaDot youtube channel with plenty of tutorials. Might as well do it all at once, the video creation shouldn't take long, as i'll be working on it from this point.

**Comment by kkukelka** [comment]
Sweet! i will implement the questions tomorrow.

- Regarding ramp: yes lets just leave it at question 6 for now
- regarding tutorials: i'll just delete the tutorial sections for now and create a followup issue that can be started once tutorials are finished
